vscode-dotty/src/extension.ts

```diff
@@ -83,7 +82,7 @@ export function activate(context: ExtensionContext) {
 
   } else {
     let configuredProject: Thenable<void> = Promise.resolve()
-    if (isUnconfiguredProject()) {
+    if (!isConfiguredProject()) {
       configuredProject = vscode.window.showInformationMessage(
         "This looks like an unconfigured Scala project. Would you like to start the Dotty IDE?",
```

Comment:
Looks like this message is still going to be shown if disableDottyIDEFile is present

Reply:
🤦‍♂️